xml/libvirt_host.xml

```diff
@@ -0,0 +1,1997 @@
+            Therefore, specify an arbitrary string here that will then be
+            ignored. The path is required, otherwise the XML parser will fail.
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </varlistentry>
+       <varlistentry>
+        <term><emphasis role="bold">Type</emphasis><guimenu>netfs</guimenu></term>
+        <listitem>
+         <itemizedlist mark="bullet" spacing="normal">
+          <listitem>
+           <para>
+            <guimenu>Target Path</guimenu>: Mount point on the &vmhost; file
+            system.
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            <guimenu>Host Name</guimenu>: IP address or host name of the server
+            exporting the network file system.
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            <guimenu>Source Path</guimenu>: Directory on the server that is
+            being exported.
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </varlistentry>
+<!-- <varlistentry>
+       <term><emphasis role="bold">Type</emphasis> <guimenu>rbd</guimenu></term>
+       <listitem>
+        <itemizedlist>
+         <listitem>
+          <para>
+           <guimenu>Host Name</guimenu>: ...
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           <guimenu>Source Name</guimenu>: ...
+          </para>
+         </listitem>
+        </itemizedlist>
+       </listitem>
+      </varlistentry>-->
+       <varlistentry>
+        <term><emphasis role="bold">Type</emphasis><guimenu>scsi</guimenu></term>
+        <listitem>
+         <itemizedlist mark="bullet" spacing="normal">
+          <listitem>
+           <para>
+            <guimenu>Target Path</guimenu>: The directory containing the device
+            file. Use <literal>/dev/disk/by-path</literal> (default) or
+            <literal>/dev/disk/by-id</literal>.
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            <guimenu>Source Path</guimenu>: Name of the SCSI adapter.
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </varlistentry>
+<!-- <varlistentry>
+       <term><emphasis role="bold">Type</emphasis> <guimenu>rbd</guimenu></term>
+       <listitem>
+        <itemizedlist>
+         <listitem>
+          <para>
+           <guimenu>Host Name</guimenu>: ...
+          </para>
+         </listitem>
+         <listitem>
+          <para>
+           <guimenu>Source Name</guimenu>: ...
+          </para>
+         </listitem>
+        </itemizedlist>
+       </listitem>
+      </varlistentry>-->
+      </variablelist>
+      <note>
+       <title>File Browsing</title>
+       <para>
+        Using the file browser by clicking <guimenu>Browse</guimenu> is not
+        possible when operating from remote.
```

Comment:
Again: when operating from **a** remote _what_?

If this is not know, then I suggest: 

... is not possible when operating remotely.